detect_secrets/audit/report.py

```diff
@@ -0,0 +1,83 @@
+import hashlib
+from enum import Enum
+from typing import Callable
+
+from ..constants import VerifiedResult
+from .common import get_all_secrets_from_file
+from .common import get_baseline_from_file
+from .common import LineGetter
+from .common import open_file
+
+
+class SecretClassToPrint(Enum):
+    REAL_SECRET = 1
+    FALSE_POSITIVE = 2
+
+    def from_class(secret_class: VerifiedResult) -> Enum:
+        if secret_class in [VerifiedResult.UNVERIFIED, VerifiedResult.VERIFIED_TRUE]:
+            return SecretClassToPrint.REAL_SECRET
+        else:
+            return SecretClassToPrint.FALSE_POSITIVE
+
+
+def generate_report(
+    baseline_file: str,
+    class_to_print: SecretClassToPrint = None,
+    line_getter_factory: Callable[[str], 'LineGetter'] = open_file,
+) -> None:
+    secrets = {}
+    for filename, secret in get_baseline_from_file(baseline_file):
+        verified_result = get_verified_result_from_boolean(secret.is_secret)
+        if class_to_print is not None and SecretClassToPrint.from_class(verified_result) != class_to_print:  # noqa: E501
```

Comment:
FWIW, you can avoid the super long line by splitting this up. e.g.

```suggestion
        if class_to_print is not None and SecretClassToPrint.from_class(verified_result) != class_to_print:  # noqa: E501
        if (
            class_to_print is not None
            and SecretClassToPrint.from_class(verified_result) != class_to_print
        ):
```